OpenMEEG/libs/OpenMEEGMaths/include/symmatrix.h

```diff
@@ -324,11 +324,11 @@ namespace OpenMEEG {
         // LU
         int *pivots = new int[nlin()];
         int Info = 0;
-        DSPTRF('U', nlin(), data(), pivots, Info);
+        DSPTRF('U', (int)nlin(), data(), pivots, Info);
         // Inverse
         int sz = (int) this->nlin() * 64;
         double *work = new double[sz];
-        DSPTRI('U', nlin(), data(), pivots, work, Info);
+        DSPTRI('U', (int)nlin(), data(), pivots, work, Info);
```

Comment:
Not sure I like all those changes. First forget about C casting, it is not safe. Then OK, you silence the warning but silently introduce a bug. If you want to follow that path, you should test that the int(s) are not negative...